prow/plugins/trigger/push.go

```diff
@@ -17,12 +17,44 @@ limitations under the License.
 package trigger
 
 import (
+	"fmt"
+	"strings"
+
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plank"
 )
 
+var mergeRe = `Automatic merge from submit-queue`
+
+func notifyOnSlackIfManualMerge(sc SlackClient, pe github.PushEvent) error {
+	if sc == nil {
+		return nil
+	}
+	sendMessage := false
+	for _, commit := range pe.Commits {
+		if strings.Contains(commit.Message, mergeRe) {
```

Comment:
I am not sure then what it means in the issue "Submit queue merge". I think that I should send message only if the author is not in the whitelist?